src/libs/OptionsListUtils.ts

```diff
@@ -1398,14 +1369,8 @@ function getOptions(
 
 /**
  * Build the options for the Search view
- *
- * @param {Object} reports
- * @param {Object} personalDetails
- * @param {String} searchValue
- * @param {Array<String>} betas
- * @returns {Object}
  */
-function getSearchOptions(reports, personalDetails, searchValue = '', betas) {
+function getSearchOptions(reports: Record<string, Report>, personalDetails: OnyxCollection<PersonalDetails>, searchValue = '', betas: Beta[] = []) {
```

Comment:
```suggestion
function getSearchOptions(reports: OnyxCollection<Report>, personalDetails: OnyxEntry<PersonalDetailsList>, searchValue = '', betas: Beta[] = []) {
```

`reports` shoulde be an Onyx collection.

PersonalDetails is not a Onyx Collection.

Reply:
And add return type!